core-web/libs/dotcms-js/src/lib/core/login.service.ts

```diff
@@ -25,8 +25,11 @@ export const LOGOUT_URL = '/dotAdmin/logout';
  * This Service get the server configuration to display in the login component
  * and execute the login and forgot password routines
  */
-@Injectable()
+@Injectable({
+    providedIn: 'root'
+})
 export class LoginService {
+    currentUserLanguageId: string = '';
```

Comment:
what about set a default one ?

Reply:
Could be us-US, but when you log in, always comes with an option.